package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "istanbul-combine",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Please do not bump the package version. I will do that when I cut another release.